### ISSUE TITLE
Working Diectory for file output

### ISSUE THREAD

**Issue by pumplerod** [reworkd/AgentGPT#372]
Where are files supposed to be written?  Can I set a specific directory for output?

Even this simple task will not actually complete...
`1) write a text file called: "test.txt" containing the text string "This is only a test"
2) show the full path to this text file
3) exit`

**Comment by benjaminbales** [comment]
my problem exactly.  I asked it to create a piece of software, but I can't see the code it produced.

**Comment by rodion-m** [comment]
It's an important question, it's really a problem to find the Agent output files. @asim-shrestha and @awtkns please help us.

**Comment by fangzhenjie** [comment]
> my problem exactly. I asked it to create a piece of software, but I can't see the code it produced.


metoo， did you solve this problem

**Comment by zjhr** [comment]
Me too. I need this feature.

**Comment by shoutian38** [comment]
Indeed. It failed to output any .txt or .docx file to any directory even when specifically asked to.

**Comment by ifoundamachine** [comment]
Same here. It seems the agent is basically pretending to have completed tasks. The created folders and documents don't exist on my hard drive.

**Comment by ivan-vilches** [comment]
Same here any news?

**Comment by xqqcylh6780** [comment]
任务提示已经完成，但是内容无法显示，请问保存在哪里了？？能不能保存至工作目录里，以txt文本或者.log以及别的文件格式

**Comment by jasangill1** [comment]
Hello @ivan-vilches @benjaminbales @pumplerod @ifoundamachine The application does not give you a file to download in the first place. Those are functions we are currently working on !! Closing due to misunderstandings.
